src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -59,6 +61,57 @@ public void postAnalysisOverview(String project, String repository, String pullR
   }
   
   /**
+   * Approve pull-request
+   */
+  public void approvePullRequest(String project, String repository, String pullRequestId, String user, StashClient stashClient){
+    try {
+      stashClient.approvePullRequest(project, repository, pullRequestId);
+      
+      LOGGER.info("Pull-request {} ({}/{}) APPROVED by user \"{}\"", pullRequestId, project, repository, user);
+      
+    } catch(StashClientException e){
+      LOGGER.error("Unable to approve pull-request: {}", e.getMessage());
+      LOGGER.debug("Exception stack trace", e);
+    }
+  }
+  
+  /**
+   * Reset pull-request approval
+   */
+  public void resetPullRequestApproval(String project, String repository, String pullRequestId, String user, StashClient stashClient){
+    try {
+      stashClient.resetPullRequestApproval(project, repository, pullRequestId);
+      
+      LOGGER.info("Pull-request {} ({}/{}) NOT APPROVED by user \"{}\"", pullRequestId, project, repository, user);
+      
+    } catch(StashClientException e){
+      LOGGER.error("Unable to reset pull-request approval: {}", e.getMessage());
+      LOGGER.debug("Exception stack trace", e);
+    }
+  }
+  
+  /**
+   * Add a reviewer to the current pull-request.
+   */
+  public void addPullRequestReviewer(String project, String repository, String pullRequestId, String user, StashClient stashClient){
+    try {
+      StashPullRequest pullRequest = stashClient.getPullRequest(project, repository, pullRequestId);
+      
+      // user not yet in reviewer list
+      StashUser reviewer = pullRequest.getReviewer(user);
+      if (reviewer == null) {
+        reviewer = stashClient.getUser(project, repository, pullRequestId, user);
+        stashClient.addPullRequestReviewer(project, repository, pullRequestId, pullRequest.getVersion(), reviewer);
+      
+        LOGGER.debug("User \"{}\" is now a reviewer of the pull-request {} #{}", user, pullRequestId, project, repository);
```

Comment:
Since all other logs are with level info, does not it make sense to have this one as well? The question is open, since it is not a "final" action.